decide/utils.hpp

```diff
@@ -5,4 +5,12 @@
 
 // Add util functions here
 
+// Calculates the area of a triangle
+double triangleArea(Coordinate p1, Coordinate p2, Coordinate p3) {
+  // See https://www.omnicalculator.com/math/area-triangle-coordinates
+  return 1 / 2 *
```

Comment:
I would wrap `1 / 2` in brackets just for clarity, otherwise this looks good.

Reply:
Turns out this was the bug. `1/2` always produces 0 due to integer division.